Open Judge System/Web/OJS.Web/Scripts/Administration/Problems/problems-edit.js

```diff
@@ -33,9 +56,9 @@
         numericTextBox.enable(true);
         input.attr("data-val-required", "Лимита е задължителен!");
 
-        $("form").removeData("validator");
-        $("form").removeData("unobtrusiveValidation");
-        $.validator.unobtrusive.parse($('form'));
+        $("#edit-form form").removeData("validator");
```

Comment:
Move removing validator in some module and use it in this and `problems-create.js` module.